eth/handler.go

```diff
@@ -74,7 +74,10 @@ type txPool interface {
 
 	// Pending should return pending transactions.
 	// The slice should be modifiable by the caller.
-	Pending(minTip *uint256.Int, baseFee *uint256.Int, blobFee *uint256.Int) map[common.Address][]*txpool.LazyTransaction
+	Pending(*uint256.Int, *uint256.Int, *uint256.Int) *txpool.PendingSuperSet
```

Comment:
fix docstring, also perhaps return the `Pending` interface instead of concrete type